test/requirements-test-gpu.txt

```diff
@@ -1,5 +1,5 @@
 -r requirements-test.txt
 auto-gptq
 autoawq
-# only available on Linux currently
-bitsandbytes==0.43.3
+bitsandbytes
+triton
```

Comment:
Do you know which test depends on triton?

Reply:
Olive doesn't explicitly have this dependency. bitsandbytes has dependency on triton.

Reply:
If it doesn't get installed automatically, we probably should add it to https://github.com/microsoft/Olive/blob/main/olive/olive_config.json#L366 as well.